### DISCUSSION TITLE
ImageFilter.Detail

### DISCUSSION THREAD

**Discussion by theoretischer** [WASasquatch/was-node-suite-comfyui#136] [Ideas]
Hi, i am trying to recreate the behavior of the A1111 loopback scaler and the last thing i need is ImageFilter.Detail. I saw you already added a lot of options from ImageFilter and I would be really thankful if you could add Detail as well!

Sorry for my bad english

**Comment by WASasquatch** [comment]
I've added a `ImageFilter.DETAIL` toggle to Image Filter Adjustments node. Image Filter Adjustments also supports batches now. 

![image](https://github.com/WASasquatch/was-node-suite-comfyui/assets/1151589/d2d73033-8ae1-4e80-918d-4f6311fd6482)
